webclient/config-local.js

```diff
@@ -6,8 +6,8 @@ var config = {
     // hash-based navigation in development builds.
     app_prefix: "",
     // Prefix for 'cdn' content, e.g. images.
-    cdn_prefix: "/cdn/",
+    cdn_prefix: "https://roomapi.tum.sexy/cdn/",
     // Prefix for API requests
-    api_prefix: "http://localhost:8080/api/",
+    api_prefix: "https://roomapi.tum.sexy/api/",
```

Comment:
It's probably a matter of preference, whether the local build should use the local server by default or the public one. I'd prefer using the local one by default, but I'm also okay changing this. However in this case we need to update the Readme then

Reply:
Oops.. that file should not have been commited